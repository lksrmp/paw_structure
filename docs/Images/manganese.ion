ION COMPLEXES
T1                0.00096755
T2                6.09558768
SNAPSHOTS                500
ID1                       MN
ID2                       O_
ID3                       H_
CUT1              3.00000000
CUT2              1.40000000
UNIT CELL     
   12.42959911     0.00000000     0.00000000
    0.00000000    12.42959911     0.00000000
    0.00000000     0.00000000    12.42959911
------------------------------------------------------------------------------------
TIME          0.00096755    ITERATION     8             ATOMS         19            
NAME          ID            INDEX                      X             Y             Z
MN0           MN            192               4.16937970    7.59571946    4.94350965
O_58          O_            174               4.60035967    6.72977952    6.82805951
O_59          O_            177               4.25807970    8.43154940    2.81777980
O_60          O_            180               4.58005967    5.63411960    3.84727972
O_61          O_            183               3.67984974    9.73934930    5.93180958
O_62          O_            186               2.13833985    6.68569952    5.21833963
O_63          O_            189               6.13070956    8.42489940    4.99978964
H_115         H_            116               3.83984973    6.93073950    7.43650947
H_116         H_            117               5.35325962    7.27647948    7.31916948
H_117         H_            118               3.29528976    8.32883940    2.65275981
H_118         H_            119               4.31129969    9.41147933    2.93269979
H_119         H_            120               5.31268962    5.00896964    3.99626971
H_120         H_            121               4.17491970    5.35489962    2.99614979
H_121         H_            122               3.06879978   10.34664926    5.47067961
H_122         H_            123               4.06181971   10.22919927    6.74239952
H_123         H_            124               1.45125990    7.37092947    5.33074962
H_124         H_            125               1.63728988    6.18947956    4.55214967
H_125         H_            126               6.64375952    8.59423938    5.83721958
H_126         H_            127               6.20374956    9.30251933    4.52758968
------------------------------------------------------------------------------------
TIME          0.01318292    ITERATION     109           ATOMS         19            
NAME          ID            INDEX                      X             Y             Z
MN0           MN            192               4.16937970    7.59571946    4.94350965
O_58          O_            174               4.60035967    6.72977952    6.82805951

...
