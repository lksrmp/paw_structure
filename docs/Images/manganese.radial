RADIAL DISTRIBUTION FUNCTION
T1                3.00002088
T2                6.09558768
SNAPSHOTS              10000
ID1                       MN
ID2                       O_
CUT               7.00000000
NBINS                   2000
RHO               0.03332795
UNIT CELL     
   12.42959911     0.00000000     0.00000000
    0.00000000    12.42959911     0.00000000
    0.00000000     0.00000000    12.42959911

        RADIUS           RDF  COORDINATION
    0.00350000     0.00000000     0.00000000
    0.00700000     0.00000000     0.00000000
    0.01050000     0.00000000     0.00000000
    0.01400000     0.00000000     0.00000000
...
    6.98950000     1.06880922    47.88294895
    6.99300000     0.89466786    47.95328415
    6.99650000     0.88958973    48.01726616
    7.00000000     0.95138275    48.08334921
