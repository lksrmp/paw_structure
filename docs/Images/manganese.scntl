!SCNTL
    ROOT_OFF        manganese
    PBC_FOLDING     TRUE

    !TRA
        T1          6.0
        T2          14.7
        N           500
        SAVE        TRUE
        LOAD        FALSE
    !END
    
    !ION
        ID1         MN
        ID2         O_
        ID3         H_
        CUT1        3.0
        CUT2        1.3
    !END
    
    !WATER
        ID1         O_
        ID2         H_
        CUT         1.4
    !END
    
    !RADIAL
        ID1     MN
        ID2     O_
        CUT     6.5
        NBINS   2000
        T1      10.0
        T2      14.7
        N       10000
    !END    
!END
