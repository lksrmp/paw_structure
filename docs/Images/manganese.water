WATER COMPLEXES
T1                0.00096755
T2                6.09558768
SNAPSHOTS                500
ID1                       O_
ID2                       H_
CUT               1.40000000
UNIT CELL     
   12.42959911     0.00000000     0.00000000
    0.00000000    12.42959911     0.00000000
    0.00000000     0.00000000    12.42959911
------------------------------------------------------------------------------------
TIME          0.00096755    ITERATION     8             ATOMS         2             
NAME          ID            INDEX                      X             Y             Z
H_64          H_            65                9.48214932    8.51605939    4.94259965
O_32          O_            148               9.88905929    9.36835933    5.26393962
------------------------------------------------------------------------------------
TIME          0.01318292    ITERATION     109           ATOMS         2             
NAME          ID            INDEX                      X             Y             Z
H_64          H_            65                9.48214932    8.51605939    4.94259965
O_32          O_            148               9.88905929    9.36835933    5.26393962
------------------------------------------------------------------------------------
TIME          0.02539828    ITERATION     210           ATOMS         2             
NAME          ID            INDEX                      X             Y             Z
H_64          H_            65                9.48214932    8.51605939    4.94259965
O_32          O_            148               9.88905929    9.36835933    5.26393962

...
