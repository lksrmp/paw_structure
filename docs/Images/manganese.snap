SELECTED SNAPSHOTS FROM TRAJECTORY FILE
T1                0.00096755
T2                6.09558768
SNAPSHOTS               2000
ATOMS                    192
UNIT CELL     
   12.42959911     0.00000000     0.00000000
    0.00000000    12.42959911     0.00000000
    0.00000000     0.00000000    12.42959911
------------------------------------------------------------------------------------
TIME          0.00096755    ITERATION     8             
NAME          ID            INDEX                      X             Y             Z
H_0           H_            1                 7.91408943    0.19515999    7.03327950
...
H_126         H_            127               6.20374956    9.30251933    4.52758968
O_7           O_            128              11.32853919    1.66152988    1.32311991
...
O_42          O_            191               0.61916996    2.76076980    4.25069970
MN0           MN            192               4.16937970    7.59571946    4.94350965
------------------------------------------------------------------------------------
TIME          0.00399116    ITERATION     33            
NAME          ID            INDEX                      X             Y             Z
H_0           H_            1                 7.91408943    0.19515999    7.03327950
...
H_126         H_            127               6.20374956    9.30251933    4.52758968
O_7           O_            128              11.32853919    1.66152988    1.32311991
...
O_42          O_            191               0.61916996    2.76076980    4.25069970
MN0           MN            192               4.16937970    7.59571946    4.94350965
------------------------------------------------------------------------------------

...
